src/invoices/InvoiceForm/InvoiceForm.js

```diff
@@ -55,6 +55,7 @@ import {
 import {
   ApprovedBy,
 } from '../../common/components';
+import { SECTIONS_INVOICE as SECTIONS } from '../constants';
```

Comment:
I don't see that `SECTIONS_INVOICE` is a new object. There are section ids for Invoice details, they might differ from the Invoice form, that's why we had `SECTIONS` here

Reply:
Invoice details have the same sections as an Invoice form, except 'Extended Information'. I don't think we need to keep the same const in two places.